backend/src/akvo/lumen/protocols.clj

```diff
@@ -46,6 +46,11 @@
     [this tenant-label]
     "List tenants users"))
 
+(defprotocol KeycloakAuthorization
```

Comment:
I'd rename the protocol name to something like `AuthoriserService` ... so don't include `Keycloak` inside the name